src/main/java/gov/cabinetoffice/gap/applybackend/dto/api/GetGrandAdvertDto.java

```diff
@@ -15,10 +15,13 @@
 public class GetGrandAdvertDto {
     private UUID id;
     private String externalSubmissionUrl;
-    private int version;
+    @Builder.Default
+    private int version = 1;
```

Comment:
wouldn't it make more sense to set this to 2 instead of remove the default value?